ingenialink/dictionary.py

```diff
@@ -134,15 +152,16 @@ def load_errors(self) -> None:
             if label is None:
                 logger.warning(f"Could not load label of error {element.attrib['id']}")
                 continue
-            self._errors[int(element.attrib["id"], 16)] = [
-                element.attrib["id"],
-                element.attrib["affected_module"],
-                element.attrib["error_type"].capitalize(),
-                label.text,
-            ]
+            error_id = int(element.attrib["id"], 16)
+            error_description = label.text
+            error_type = element.attrib["error_type"].capitalize()
```

Comment:
Parsers should not modify the values of the things they parse.
@GullinBustin Do you know if error type being used anywhere?

Reply:
The error type is used in the error widget of ML3 for the category field.